### PR TITLE
Created Dialog and Made Spelling Fixes

### PR DESCRIPTION
**Pull request by GostOmega**
Gave Dialog too users who register with the pokedex. I'll be moved over to the command area for !pokemon register on master branch.
In general making the bot seem more universal across servers with the (GREETING).format (message can be changed later)
Fixed few spelling issues and updated role from Trainer to '@ Trainer'

UPDATE: no longer using the (GREETING).format but still using the dialog.

### CODE REVIEW

**Comment by TRAINer** [comment]
Can I be deleted from this thread?
I'm not developing this software.

Yevgeny Gromov
20.04.2016 21:58 пользователь "GostOmega" <notifications@github.com>
написал:

File: KantoBot.py [hunk 1]
```diff
@@ -71,7 +75,7 @@ def __init__(self, bot):
 
     print("Creating kantobot info for user " + user.name)
 
-    # Create the base trainer role iff it doesn't exist
+    # Create the base trainer role if it doesn't exist
```
Comment (ed-george) [inline KantoBot.py]: This is intentional. Iff, with the double f, means [if and only if](https://en.wikipedia.org/wiki/If_and_only_if)

File: KantoBot.py [hunk 2]
```diff
@@ -92,9 +96,11 @@ def __init__(self, bot):
         # Add role to user
         await self.bot.add_roles(user, role)
         print("Gave " + BASE_ROLE_NAME + " role to " + user.name)
+        await self.bot.say(GREETTING).format(server.name)
```
Comment (ed-george) [inline KantoBot.py]: There seems to be a spelling mistake here

**Comment by ed-george** [comment]
Thanks @GostOmega, can you confirm this has been tested? I cannot seem to get any `.format()` calls working with the `say` command.

**Pull request closed without merging**